packages/wagmi/hooks/useSendTransaction.ts

```diff
@@ -15,9 +16,11 @@ export function useSendTransaction({
   onSettled,
   prepare,
   enabled = true,
+  gasMargin = false,
```

Comment:
Seems like this is something we always want. Unsure when we wouldn't want a gas margin?